sk0-front/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:20-alpine AS build
+WORKDIR /app
```

Comment:
on my mac works only if between FROM and workdir you place a new line: 
```RUN apk add --no-cache python3 make g++```
- @markkovari  any idea why?

Reply:
another simple solution to change the initial image